### PR TITLE
PLAT-143436: Fixed Panel and Tabgroup layout for RTL locales

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Some paddings for Panel and Tabgroup were not adapted for RTL locales.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Adapted paddings for RTL

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-143436

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM. Nice catch 🐟

**Review by MikyungKim — COMMENTED** [review]
Is TabGroup issue reproducible now?

**Comment by daniel-stoian-lgp** [comment]
> Is TabGroup issue reproducible now?

Hello @MikyungKim ,
On my opinion, it is still reproductible

Electro skin,  orientation vertical, LTR locale
image

RTL locale:

image

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 ### Fixed
 
 - `agate/ContextualPopupDecorator` layout for Carbon, Cobalt, Copper, Electro, Titanium skins
+- `agate/Panel` and `agate/Tabgroup` padding for RTL locales
```
Comment (MikyungKim) [inline CHANGELOG.md:10]:
```suggestion
- `agate/Panel` and `agate/TabGroup` padding for RTL locales
```

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**